### PR TITLE
Shell app and admin refactor

### PR DESCRIPTION
**Pull request by zovomat**

### CODE REVIEW

**Review by giuliano176 — DISMISSED** [review]

**Review by giuliano176 — DISMISSED** [review]

**Review by nubsthead — DISMISSED** [review]

File: src/boot/app/app-loader-mounter.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
```
Comment (geeky-abhishek) [inline src/boot/app/app-loader-mounter.tsx:3]: In one file I have seen Year **2022** .Which one is correct 2021 or 2022 ?

File: src/boot/app/load-apps.ts [hunk 2]
```diff
@@ -6,28 +6,30 @@
...
-export function loadApps(storeFactory: StoreFactory, apps: Array<AppData>): void {
+export function loadApps(storeFactory: StoreFactory, apps: Array<CarbonioModule>): void {
 	injectSharedLibraries();
-	const appsToLoad =
-		typeof cliSettings === 'undefined' || cliSettings.enableErrorReporter
-			? apps
-			: filter(apps, (app) => app.core.name !== 'carbonio-error-reporter-ui');
+	const appsToLoad = filter(apps, (app) => {
+		if (app.name === SHELL_APP_ID) return false;
+		if (app.attrKey && getUserSetting('attrs', app.attrKey) !== 'TRUE') return false;
```
Comment (geeky-abhishek) [inline src/boot/app/load-apps.ts:24]: can't we combine both if using '||' OR ?

**Review by geeky-abhishek — DISMISSED** [review]

File: src/boot/app/app-loader-mounter.tsx (hunk 1 above)
Comment (zovomat) [reply src/boot/app/app-loader-mounter.tsx:3]: both, it's the year of creation of the file

**Review by dhavaldodiya — DISMISSED** [review]

File: src/boot/app/load-apps.ts (hunk 2 above)
Comment (zovomat) [reply src/boot/app/load-apps.ts:24]: it's a bit redundant but i find it more readable because the two conditions are unrelated and subject to change

**Review by geeky-abhishek — APPROVED** [review]

**Review by giuliano176 — APPROVED** [review]

**Review by giuliano176 — APPROVED** [review]

**Pull request merged**